libsql-replication/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+pub mod frame;
+mod injector;
+pub mod meta;
+pub mod replicator;
+pub mod rpc;
+pub mod snapshot;
+
+mod error;
+
+pub const LIBSQL_PAGE_SIZE: usize = 4096;
+
+#[cfg(test)]
+pub mod test {
+    use arbitrary::Unstructured;
+    use bytes::Bytes;
+
+    /// generate an arbitrary rpc value. see build.rs for usage.
```

Comment:
I looked at build.rs and I still have no clue what is this for :sweat_smile: But I suppose it is only needed for some specific tests? What's the reason the commented-out code doesn't work and why is it fine to leave it as `todo!()`?

Reply:
This is for generating arbitrary values for property testing. Those function were used to derive the arbitrary trait on RPC types. I'm disabling that for now because it meant moving even more code in replication that I didn't want there, and this is going away with the streaming proxy in a follow-up